### DISCUSSION TITLE
Ligthbox without HTML

### DISCUSSION THREAD

**Discussion by amargoCactus** [sachinchoolur/lightGallery#1294] [Q&A]
I want to generate a lightbox without having to make an html, just pass the url of the images to be loaded to the instance, is it possible to do it that way?

**Comment by Lex-2008** [comment]
I think it's what dynamic mode is for:
demo: https://www.lightgalleryjs.com/demos/dynamic-mode/
docs: https://www.lightgalleryjs.com/docs/dynamic-variables/
